packages/facility-server/__tests__/apiv1/User.test.js

```diff
@@ -340,108 +355,144 @@ describe('User', () => {
   });
 
   describe('User facility methods', () => {
-    let user = null;
     let superUser = null;
-    let noFacilityUser = null;
+    let linkedUser = null;
+    let unlinkedUser = null;
 
-    const validUserFacilities = [facility1, facility2];
-    const validUserFacilityIds = validUserFacilities.map((f) => f.id);
+    const nonSensitiveFacilities = [facility1, facility2, facility3];
+    const nonSensitiveFacilityIds = nonSensitiveFacilities.map(f => f.id);
 
-    beforeAll(async () => {
+    const linkedUserFacilities = [facility1, sensitiveFacility1];
+    const linkedUserFacilityIds = linkedUserFacilities.map(f => f.id);
```

Comment:
Again not a fan of `linked` (I know you've just taken it and run with it, that descriptor already existed). Additionally, when these refer to facilities, calling then a name that includes `userFacilities` is a bit confusing because there's another database table that is very involved in this area which is called `user_facilities`.

```suggestion
    const user1Facilities = [facility1, sensitiveFacility1];
    const user1FacilityIds = linkedUserFacilities.map(f => f.id);
```

Reply:
wait `user_facilities` **IS** the table that we are getting this list from here? So isn't it suitable to have the name include `userFacilities`?

Reply:
Ok now I am confused 😆 where else would it come from? I thought the entire point of the `user_facilities` table was to link users to facilities for login restrictions?

<img width="337" height="152" alt="image" src="https://github.com/user-attachments/assets/b3553f21-0fbc-49f7-8f98-d67bb40f8d12" />

What am i missing?

Reply:
Yeah, the name of the array should just say what's inside it